docs/src/quickstart.md

```diff
@@ -160,6 +161,51 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+In the above `from_mcmcchains` call, we did not specify the `prior`, `posterior_predictive`, nor the `log_likelihoods`. This section describes how one would go about obtaining these from the `Model` and `Chains`, which we can then pass to `from_mcmcchains` to construct the [`InferenceData`](@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+Obtaining the predictive posterior is a matter of first instantiating a "predictive model", i.e. a `Model` but with the observations set to `missing`, and then call `predict` on the predictive model and the inferred posterior:
```

Comment:
We can use the same approach to draw from the prior predictive distribution, right?
Also, could we be explicit that the user isn't getting the distribution but a sample from the distribution?
Maybe something like
```suggestion
To draw from the prior and posterior predictive distributions we can instantiate a "predictive model", i.e. a Turing model but with the observations set to `missing`, and then calling `predict` on the predictive model and the previously drawn samples:
```

Reply:
Yep! And I prefer your suggestion, so let's go with that:)